tests/src/test/scala/org/apache/openwhisk/core/database/test/ReplicatorTests.scala

```diff
@@ -350,13 +350,18 @@ class ReplicatorTests
   it should "continuously update a database" in {
     // Create a database to backup
     val dbName = testDbPrefix + "database_for_continous_replication"
+    val backupDbName = s"continuous_$dbName"
+
+    // Pre-test cleanup of previously created entities
+    removeDatabase(backupDbName, true)
```

Comment:
Just need to remove entities related to `backupDbName`. The `createDatabase()` method attempts to clean `dbName` before creating the database.